src/TwigComponent/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 2.23.0
+
+- Add `ComponentPropertiesExtractor` to extract component properties from a Twig component
```

Comment:
I don't think we want to expose this class (with BC promise) for something like this. At least in a first step.

Reply:
No, my point is i'm not sure we should make this class "public" for now. Because once it's done, we cannot go back. 

And, as i said in the previous message, we need to clarifiy first whose class has which responsibility, the contracts, etc.